genome_kit/genome_track.py

```diff
@@ -76,6 +76,10 @@ def __init__(self, outfile, etype, strandedness, reference_genome, dim=1, resolu
             * ``"single_stranded"``: both strands share the same data. The data is applied in Interval coordinate (reference strand) order.
             * ``"strand_unaware"``: ignores the Interval strand, data is applied in Interval coordinate (reference strand) order.
             * ``"strand_aware"``: data is applied from 5" end to 3" end (sense strand order).
+
+            Negative strand data from bedgraph or wig files must be provided from a separate file. When calling
+            ``set_data_from_bedgraph()`` or ``set_data_from_wig()``, set ``"strand_unaware"`` (``"single_stranded"`` is
+            equivalent when only providing positive strand data).
```

Comment:
seems unnecessarily wordy, is the recommendation using two different enums in two scenarios or to always use strand_unware?

Reply:
You're right, I'll remove that last part. The recommendation is to always use strand_unaware.